verifier/verifier.go

```diff
@@ -95,7 +103,7 @@ func (v *verifier) Verify(ctx context.Context, desc ocispec.Descriptor, signatur
 	return outcome, outcome.Error
 }
 
-func (v *verifier) processSignature(ctx context.Context, sigBlob []byte, envelopeMediaType string, trustPolicy *trustpolicy.TrustPolicy, pluginConfig map[string]string, outcome *notation.VerificationOutcome) error {
+func (v *verifier) processSignature(ctx context.Context, sigBlob []byte, envelopeMediaType string, trustPolicy *trustpolicy.TrustPolicy, x509TrustStore truststore.X509TrustStore, pluginConfig map[string]string, outcome *notation.VerificationOutcome) error {
```

Comment:
We don't need to pass the trust store here, do we?

Reply:
Oh yeah, we have v.trustStore now.